### ISSUE TITLE
Failed to add the problem to favorite. Please open the output channel for details.

### ISSUE THREAD

**Issue by codingYH** [LeetCode-OpenSource/vscode-leetcode#617]
node:57237) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
[ERROR] http error [code=520]

node -v
v14.7.0

**Comment by yihong0618** [comment]
@codingYH 
Yes the `add the problem to favorite` seems expired in leetcode and I checked in leetcode-cn works well.
Let me try to fix it.

**Comment by codingYH** [comment]
Thanks a lot! Are the company tag questions the same with Leetcode?

**Comment by codingYH** [comment]
> @codingYH
> Yes the `add the problem to favorite` seems expired in leetcode and I checked in leetcode-cn works well.
> Let me try to fix it.

I found the data was successfully posted to leetcode.com. However, in Visual Studio, It didn't work.

**Comment by yihong0618** [comment]
@jdneo 
Change api to graphql seems can solve this problem. And Seems leetcode is dropping the old apis bit by bit.

**Comment by jdneo** [comment]
@yihong0618 Cool, do you have the interest to update the cli?

**Comment by yihong0618** [comment]
@jdneo 
Doing it right now, I will send a pr later.

**Comment by yihong0618** [comment]
@jdneo https://github.com/yihong0618/leetcode-cli/commit/c0f9524b6939f81d9b8ec367221a83f8cc5444b8  fixed

**Comment by codingYH** [comment]
There is the same bug. Failed to add the problem to favorite.

**Comment by codingYH** [comment]
I think the bug is still here.

**Comment by niharika2k00** [comment]
Failed to add the problem to favorite.
